modules/transport/index.js

```diff
@@ -0,0 +1,59 @@
+// FIXME: change to '../API' when we update to webpack2. If we do this now all
+// files from API modules will be included in external_api.js.
+import { API_ID } from '../API/constants';
+import { getJitsiMeetGlobalNS } from '../util/helpers';
+
+import Transport from './Transport';
+import PostMessageTransportBackend from './PostMessageTransportBackend';
+
+export {
+    Transport,
+    PostMessageTransportBackend
+};
+
+/**
+ * Option for the default low level transport.
+ *
+ * @type {Object}
+ */
+const postisOptions = {};
+
+if (typeof API_ID === 'number') {
+    postisOptions.scope = `jitsi_meet_external_api_${API_ID}`;
```

Comment:
I don't have we have problems with the format when external_api.js is used! 

We have problems with the format if somebody is listening/sending directly from the iframe's window object. But maybe we can detect this one as well.

Thanks for pointing this! We can discuss it more offline!